src/WebApi_inverter.cpp

```diff
@@ -245,6 +249,8 @@ void WebApiInverterClass::onInverterEdit(AsyncWebServerRequest* request)
     inverter.Serial = new_serial;
     strncpy(inverter.Name, root[F("name")].as<String>().c_str(), INV_MAX_NAME_STRLEN);
 
+    inverter.CurrentPhase = strtoll(root[F("phase")].as<String>().c_str(), NULL, 16);
```

Comment:
Why do you convert the phase value to a string first and then to a long long number? Convert it to a uint8_t immediatly to prevent unnecessary String constructor calls

Reply:
I have no glue to this line. I made a copy paste of the line 245. 
How should it look like?

Reply:
Just write: `root[F("phase")].as<uint8_t>()`

Reply:
Done.